app/components/dialog/components/collection.tsx

```diff
@@ -35,6 +36,7 @@ interface CollectionDialogStates {
 class CollectionDialog extends React.PureComponent<CollectionDialogProps, CollectionDialogStates> {
   private contentBox: HTMLDivElement | null;
   private newCollectionAnchor: HTMLDivElement | null;
+  private cancelToken = Axios.CancelToken.source().token;
```

Comment:
If you make `cancelToken` in a component, you should call `cancel` when a component is unmounted or closed(if it is a dialog.)